Sources/Examples/DispatchQueue+PendingWork.swift

```diff
@@ -0,0 +1,12 @@
+import Dispatch
+
+extension DispatchQueue {
+	/// Returns once any pending work has been completed.
+	func pendingWorkComplete() async {
+		await withUnsafeContinuation { continuation in
```

Comment:
Please always show `withCheckedContinuation` 

```suggestion
		await withCheckedContinuation { continuation in
```

unless we're specifically discussing a reason to use the unsafe one

Reply:
I have no idea why I did this. I don't think I have ever used the usafe variants before...

Reply:
Umm, so funny story...

https://github.com/apple/swift/issues/74206

Reply:
I added an in-code TODO, because I cannot currently fix this.